stories/utils/pathd.js

```diff
@@ -0,0 +1,3 @@
+export const SETTINGS = 'M542.2-29.2l-2.3-.8-9.1-21.1 1-2.2c27.7-60.7 25.7-62.5 21.1-66.9L516-155c-1.5-1.5-4.1-2.4-6.4-2.4-2.1 0-8.4 0-62.7 23.8l-2.2 1-22.4-8.7-.8-2.2C395.8-205 393.2-205 386.8-205h-51.9c-6.3 0-9.2 0-32.9 61.5l-.8 2.3-22.3 8.9-2.2-.8c-36.8-15-58.2-22.7-63.8-22.7-2.3 0-5 1-6.4 2.4l-36.9 34.9c-4.6 4.5-6.6 6.3 22.4 65.8l1.1 2.2-9.2 21.1-2.2.8c-64.8 24.3-64.8 26.7-64.8 33v49.4c0 6.3 0 9 64.9 31.5l2.3.8 9.2 21-1 2.1c-27.5 60.7-25.8 62.4-21.1 66.9l36.8 34.8c1.6 1.5 4.2 2.4 6.4 2.4 2.1 0 8.4 0 62.7-23.8l2.2-1 22.4 8.9.8 2.2c25.6 61.4 28.1 61.4 34.6 61.4H389c6.6 0 9.2 0 32.9-61.5l1-2.3 22.3-8.7 2.1.8c36.8 15 58.2 22.7 63.8 22.7 2.3 0 5-1 6.4-2.4l36.9-34.9c4.6-4.5 6.6-6.3-22.6-65.6l-1.1-2.2 9.1-21 2.2-.8c65-24.5 65-26.9 65-33.2V2.5c.1-6.5.1-9.2-64.8-31.7zM362 108.1c-46.5 0-84.2-35.9-84.2-80.1s37.9-80.1 84.2-80.1 84.2 35.9 84.2 80.1-37.8 80.1-84.2 80.1z'
```

Comment:
I feel, its better to have independent setting, because for the scenario like this, user 1 & user 2 want have same icon with different viewbox size. if we map together, we need to maintain two settings for same icon. Let us know your thoughts, we can change accordingly.